digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/MassTransitStopDao.scala

```diff
@@ -633,6 +637,10 @@ class MassTransitStopDao {
     query + s" where a.external_id = $nationalId"
   }
 
+  def withNationalIdAndNotExpired(nationalId: Long)(query: String): String = {
+    query + s" where a.external_id = $nationalId and (a.valid_to > sysdate or a.valid_to is null)"
+  }
+
```

Comment:
Instead of creating a new method why we don't transform the method above to have the possibility to return only the not expired records.

Reply:
This should be deleted, not in use anymore since the change of course of the implementation.

Reply:
@marquesfm This part was deleted